README.md

```diff
@@ -170,12 +170,42 @@ To start the trento agent:
 > recommendations on cluster settings and state for many HA components. New rules
 > can be created and tuned to adjust to different requirements.
 
+## Trento Runner
+
+Trento Runner is responsible for running the health checks. It is based on [Ansible](https://docs.ansible.com/ansible/latest/index.html) and [ARA](https://ara.recordsansible.org/).
+This component can be executed in the same machine as the Web UI, but it is not mandatory, it can be executed in any other machine that has network access to the agents.
+
+In order to start it, some packages must be installed and started. Here a quick go through:
+```shell
+# Install ansible. Tested with version 2.11.2
+pip install ansible
+# Install ARA
+pip install --user ansible "ara[server]"
+# Setup ARA database
+ara-manage migrate
+# Start ARA server. This process can be started in background or in other shell terminal
+ara-manage runserver ip:port
+
+# Enable the SSH connections to the agents machines
+# Update the /etc/hosts file to identify the agents hosts
+# Authorize SSH connection for all the agents hosts
```

Comment:
I am a bit confused about this part, even if it works fine by adding the entries to the /etc/hosts files I wouldn't ask the user to do so. We already have a service discovery in place, we might want to use that. The consul server DNS should be  possibly used to resolve the nodes.

Reply:
The code updated to work without editing the host file.
I will update the comments here to make sure that the unique mandatory part is the ssh authorization